ui/main/src/app/services/translation/TranslationService.ts

```
@@ -11,34 +11,40 @@ import {Observable, Subject} from 'rxjs';
+    }
+
+    public static setLang(lang: string) {
+        this.translationLib.setLang(lang);
+    }
+
+    public static getTranslation(key: string, params?: Object): string {
+        return this.translationLib.getTranslation(key, params);
     }
 
     public static initLocale() {
         this.destroy$.next(); // unsubscribe from previous subscription , only useful for unit tests as we init more than one time
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)